invenio_stats/aggregations.py

```diff
@@ -270,56 +274,69 @@ def doc_type(self):
         return get_doctype('{0}-{1}-aggregation'.format(
             self.event, self.interval))
 
-    def agg_iter(self, lower_limit, upper_limit):
+    def _split_date_range(self, lower_limit, upper_limit):
```

Comment:
minor: missing docstring

Reply:
+1 to this. IIUC it builds a dict with the k,v = date in strf representation, date